src/cascadia/Remoting/Monarch.idl

```diff
@@ -7,11 +7,15 @@ namespace Microsoft.Terminal.Remoting
 {
 
     [default_interface] runtimeclass FindTargetWindowArgs {
-        CommandlineArgs Args;
+        FindTargetWindowArgs(CommandlineArgs args);
+
+        CommandlineArgs Args { get; };
         Int32 ResultTargetWindow;
     }
 
     [default_interface] runtimeclass ProposeCommandlineResult {
+        ProposeCommandlineResult(Boolean shouldCreateWindow);
+

```

Comment:
I think you don't actually need to add the constructor to the IDL (and introduce a factory, by extension) if you're only constructing these classes using `winrt::make` and `winrt::make_self`. These constructors are only used if you're trying to construct the objects when you don't have access to the impl type.